orchestration/pom.xml

```diff
@@ -118,54 +118,73 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- Additional dependencies for OSS code generator resttemplate  -->
+    <dependency>
+      <groupId>org.openapitools</groupId>
+      <artifactId>jackson-databind-nullable</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
```

Comment:
(Question)

What do we need this dependency for?

Reply:
We cannot remove the annotation with just using configuration properties. We could provide a custom mustache file. But at this point, we can stop the additional effort. It's considered a temporary solution anyway.